controllers/front/StoresController.php

```diff
@@ -126,21 +110,20 @@ public function getStoresForXml()
      */
     protected function displayAjax()
     {
-        // StarterTheme: Remove method when google maps v3 is done
         $stores = $this->getStoresForXml();
         $parnode = new SimpleXMLElement('<?xml version="1.0" encoding="UTF-8"?><markers></markers>');
 
         foreach ($stores as $store) {
-            $other = '';
             $newnode = $parnode->addChild('marker');
             $newnode->addAttribute('name', $store['name']);
             $address = $this->processStoreAddress($store);
 
-            //$other .= $this->renderStoreWorkingHours($store);
             $newnode->addAttribute('addressNoHtml', strip_tags(str_replace('<br />', ' ', $address)));
             $newnode->addAttribute('address', $address);
-            $newnode->addAttribute('other', $other);
+            $newnode->addAttribute('hours', trim($store['hours']));
             $newnode->addAttribute('phone', $store['phone']);
+            $newnode->addAttribute('fax', $store['fax']);
+            $newnode->addAttribute('note', $store['note']);
```

Comment:
So you handle hours, fax, note, but remove the processing of `$other` ?

Reply:
`$other` was never used

Reply:
OK then 👍